geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/InternalLocatorIntegrationTest.java

```diff
@@ -69,6 +70,8 @@
 
   @Before
   public void setUp() throws IOException {
+    // set a property to tell membership to create a new cluster
+    System.setProperty(GMSJoinLeave.BYPASS_DISCOVERY_PROPERTY, "true");
```

Comment:
I assume this is done to speed up the test. I like it!

But shouldn't we be restoring the system property after the test runs? I believe we have a JUnit rule for that?

Reply:
I just noticed that @bschuchardt did add a `RestoreSystemProperties` rule so this is all good now